releasenotes/notes/add-db-system-tag-redis-4a67baedfa7c2ade.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    tracing: The "db.system" tag is reported with value "redis" by redis integrations: aioredis, aredis,
+    redis, rediscluster, yaaredis.
```

Should we even add a release note? Would users want/need to know about this tag?

```suggestion
    tracing: Adds ``db.system`` span tag to the following integrations: aioredis, aredis,
    redis, rediscluster, yaaredis.
```

---

I wondered as well. My reasoning was that since I’m splitting PRs to make diffs manageable, I want to be able to track which integrations report the new tag.

---

Then again, this means I'll create a lot of noise in release notes since I'm splitting in 5 PRs or more... The tag is useful, not crucial, so I'm happy with no release note.

---

This is a user-facing change, so users may be confused without a release note if all of a sudden they see new `db.system` tags in their traces. We can alternatively have one release note that encompasses all of these split PRs 😄